modules/creating-multiple-ingress-through-single-alb.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/multiple-ingress-through-single-alb.adoc
+
+:_content-type: PROCEDURE
+[id="nw-creating-multiple-ingress-through-single-alb_{context}"]
+= Creating multiple ingresses through single ALB
+
+The user can route the traffic to multiple Ingresses through a single ALB.
+
+.Prerequisites
+
+.Procedure
+
+. Create an `AWSLoadBalancerController` and three `Ingress` resources:
++
+[source,yaml]
+----
+kind: Ingress
+metadata:
+  name: example-1 <1>
+  annotations:
+    alb.ingress.kubernetes.io/scheme: internet-facing <2>
+    alb.ingress.kubernetes.io/group.name: example <3>
```

Comment:
I believe this annotation was disabled by default: [here](https://github.com/alebedev87/aws-load-balancer-operator/blob/main/pkg/controllers/awsloadbalancercontroller/deployment.go#L208).
cc @arjunrn

Reply:
@xenolinux : well, I think that won't be enough as the chapter explains how to put multiple ingresses under the single ALB. Without `group.name` annotation, they all will be considered as separate groups resulting in 3 ALBs.

To achieve what's described in the title without this annotation we'd have to use `IngressClassParams` CR attached to the ingress class used by these Ingresses. WDYT @arjunrn ?

Reply:
I've rewritten the instructions for this tutorial over [here](https://gist.github.com/arjunrn/5fded425bab7bae90ae9896bb3f1b87e). @xenolinux can you update the instructions accordingly.

@alebedev87 I've added an [e2e test](https://github.com/openshift/aws-load-balancer-operator/pull/62) for this scenario. PTAL

Reply:
Ack'ed- updated the instructions